controllers/counsels.controller.js

```diff
@@ -0,0 +1,19 @@
+const createError = require("http-errors");
+
+const Counsel = require("../models/Counsel");
+const { RESPONSE } = require("../constants");
+
+exports.createCounsel = async (req, res, next) => {
+  try {
+    const newCounsel = req.body;
+
+    await Counsel.create(newCounsel);
```

Comment:
```suggestion
    const { title, ..... }  = req.body;
    
    const newCounsel = {
       title,
       ....
    };

    await Counsel.create(newCounsel);
```
구조분해할당을 한 번하고 새로 객체를 만들어서 전달하는 건 어떨까요? req.body에 담겨있는 정보가 어떤 정보들인지
확인할 수도 있고 예상하지 않은 인풋들을 방어해줄 수 있을 것 같아요!

Reply:
훨씬 명시적이고 좋을 것 같네요!!